packages/edit-site/src/components/sidebar-edit-mode/navigation-item-editor/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import { FlexBlock, Flex } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { navigation } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import DefaultSidebar from '../default-sidebar';
+
+export default function NavigationItemSidebar() {
```

Comment:
Not sure what this file is...? Does it need to be removed?

Reply:
Yeah, this is a bad carryover from an earlier attempt to play around with the InspectorControls. Removing it